mcu_proxy.go

```diff
@@ -144,6 +144,10 @@ func (p *mcuProxyPublisher) HasMedia(mt MediaType) bool {
 	return (p.mediaTypes & mt) == mt
 }
 
+func (p *mcuProxyPublisher) SetMedia(mt MediaType) {
+	p.mediaTypes = mt
```

Comment:
The `mediaTypes` are sent to the proxy and stored there when the publisher is created: https://github.com/strukturag/nextcloud-spreed-signaling/blob/e5f2d4d3cf387eb749f89c544d85096fc68808a0/mcu_proxy.go#L971

So this method should update them on the proxy, too.

However as the value is not really used in the proxy right now, it would be fine to add a `TODO: Also update mediaTypes on proxy.` comment here. I'm thinking the proxy doesn't need the mediatypes at all, so they could be removed from the creation call. If necessary, the proxy can parse the SDP and extract it from there.

Reply:
Done, but please check that I added the TODO in the right place ;-)